assets/constants.ts

```diff
@@ -6173,6 +6173,9 @@ export default {
   // Maximum allowed length of a state name.
   "MAX_STATE_NAME_LENGTH": 50,
 
+  // Maximum allowed length of unique progress url ID.
+  "MAX_PID_LENGTH": 6,
```

Comment:
Can we use the expanded name in the constant? PID seems too generic, it is not immediately clear what it represents without reading the comment.

Reply:
Done.

Reply:
Done.